<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft JhengHei</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>FileSearcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/file/images/logo.png</normaloff>:/file/images/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTreeView" name="tree">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="statusTip">
       <string>the file system tree</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="indentation">
       <number>10</number>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop">
          <property name="text">
           <string>stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchedit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft New Tai Lue</family>
           </font>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="classlist">
          <property name="font">
           <font>
            <family>Microsoft JhengHei</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>All Files</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/file/images/folder.png</normaloff>:/file/images/folder.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Document</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/file/images/doc.png</normaloff>:/file/images/doc.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Music</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/file/images/music.png</normaloff>:/file/images/music.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Picture</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/file/images/picture.png</normaloff>:/file/images/picture.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Video</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/file/images/video.png</normaloff>:/file/images/video.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Application</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/file/images/setting.png</normaloff>:/file/images/setting.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Others</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/file/images/andoc.png</normaloff>:/file/images/andoc.png</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableView" name="fileinfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft JhengHei</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="autoScroll">
           <bool>false</bool>
          </property>
          <property name="autoScrollMargin">
           <number>10</number>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="stretch">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>6</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>7</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextOnly</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textdetail">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="statusTip">
       <string>give a stretch of the file</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::WidgetWidth</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="collapse">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="m_new_index"/>
    <addaction name="m_update"/>
    <addaction name="separator"/>
    <addaction name="m_exit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>edit</string>
    </property>
    <addaction name="m_cut"/>
    <addaction name="m_copy"/>
    <addaction name="m_paste"/>
    <addaction name="m_all"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>option</string>
    </property>
    <addaction name="m_normal_mode"/>
    <addaction name="m_matchall"/>
    <addaction name="m_wildcard"/>
    <addaction name="m_regex"/>
    <addaction name="separator"/>
    <addaction name="m_case_sensitive"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="m_help"/>
    <addaction name="m_about"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>setting</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>language</string>
     </property>
     <addaction name="m_English"/>
     <addaction name="m_Chinese"/>
    </widget>
    <widget class="QMenu" name="menuAssign_directory">
     <property name="title">
      <string>assign directory</string>
     </property>
     <addaction name="m_all_volume"/>
     <addaction name="m_select_directory"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="menuAssign_directory"/>
    <addaction name="m_style"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
   <addaction name="menuSetting"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="m_new_index">
   <property name="text">
    <string>new index</string>
   </property>
   <property name="statusTip">
    <string>create a new index</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="m_update">
   <property name="text">
    <string>update index</string>
   </property>
   <property name="statusTip">
    <string>update the index,if you have a index already,it is much faster</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="m_cut">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>cut</string>
   </property>
   <property name="statusTip">
    <string>cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="m_copy">
   <property name="text">
    <string>copy</string>
   </property>
   <property name="statusTip">
    <string>copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="m_paste">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>paste</string>
   </property>
   <property name="statusTip">
    <string>paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="m_all">
   <property name="text">
    <string>select all</string>
   </property>
   <property name="statusTip">
    <string>select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="m_set">
   <property name="text">
    <string>setting</string>
   </property>
  </action>
  <action name="m_help">
   <property name="text">
    <string>help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="m_about">
   <property name="text">
    <string>about...</string>
   </property>
  </action>
  <action name="m_matchall">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>match whole word</string>
   </property>
   <property name="statusTip">
    <string>just match the whole word</string>
   </property>
  </action>
  <action name="m_wildcard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>use wildcard</string>
   </property>
   <property name="statusTip">
    <string>match directory when search,is is closed by default</string>
   </property>
  </action>
  <action name="m_case_sensitive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>case sensitive</string>
   </property>
   <property name="statusTip">
    <string>dont't ignore the case of the word,it is closed by default</string>
   </property>
  </action>
  <action name="m_export">
   <property name="text">
    <string>export...</string>
   </property>
  </action>
  <action name="m_exit">
   <property name="text">
    <string>exit...</string>
   </property>
   <property name="statusTip">
    <string>exit the application</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="m_regex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>regex expression</string>
   </property>
   <property name="statusTip">
    <string>use the regular expression to search,it is closed by default</string>
   </property>
  </action>
  <action name="m_English">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="m_Chinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中文-简体</string>
   </property>
  </action>
  <action name="m_normal_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>normal mode</string>
   </property>
  </action>
  <action name="actionContact_us">
   <property name="text">
    <string>Contact us</string>
   </property>
  </action>
  <action name="m_all_volume">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All Volume</string>
   </property>
  </action>
  <action name="m_select_directory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select...</string>
   </property>
  </action>
  <action name="m_style">
   <property name="text">
    <string>style</string>
   </property>
   <property name="toolTip">
    <string>choose style</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
